java/apirag-server/src/main/java/com/datasqrl/ai/DataVizFunction.java

```diff
@@ -6,11 +6,11 @@
 
 @AllArgsConstructor
 @Getter
-public enum PlotFunction {
+public enum DataVizFunction {
```

Comment:
Cal me oldschool, but I prefer no abbreviations in Class names :) `DataVisualizationFunction` is not too long.

Reply:
Done